docs/documentation/release-notes.md

```diff
@@ -207,7 +207,7 @@ In some cases, a new version is introduced which breaks backward compatibility.
 From Cortex XSOAR version 6.8 and above, there is support to mark a new version as a **breaking changes version**.
 
 Marking a version as a **breaking changes version** provides the user with an alert before installation:
-![image](../doc_imgs/integrations/bc_alert_example.png)
+![image](/doc_imgs/integrations/bc_alert_example.png)
```

Comment:
Why is this change necessary?

Reply:
I think this occurred in one of the merges from `master`, like [this one](https://github.com/demisto/content-docs/pull/1103/commits/1021fbb5f0cdfb820f4a8628a48b5c7184c89f74). From what I saw, the build actually automatically corrects the paths.

I see that the image loads fine in https://xsoar-pan-dev--pull-request-1103-mma9v2uu.web.app/docs/documentation/release-notes. Should I revert or leave as is?